README.md

```diff
@@ -34,10 +34,13 @@ Install `fixed-data-table-2` using npm.
 ```shell
 npm install fixed-data-table-2
 ```
-Add the default stylesheet `dist/fixed-data-table.css`, then import it into any module.
+Add the default stylesheet `dist/fixed-data-table.css` using a link tag or import it with a CSS module.
 
 ### Basic Example
-
+Note this example demonstrates typical FDT usage, but won't work out of the box.  
```

Comment:
Sure, do you think we should pare it down but leave some pointers to the pre-requisite knowledge?  Anything stand out as important vs can be removed?